js/app.js

```diff
@@ -19,7 +13,37 @@ answer1 = askAgain(answer1);
 console.log('Q1 Answer: ', answer1);
 question1(answer1);
 
-function question1(){
+let answer2 = prompt('Do you think I\'ve ever skydived before? Yes or No or y/n.').toLowerCase();
+answer2 = askAgain(answer2);
+console.log('Q2 Answer: ', answer2);
+question2(answer2);
+
+let answer3 = prompt('Do you think I\'ve ever been published? Yes or No or y/n.').toLowerCase();
+answer3 = askAgain(answer3);
+console.log('Q3 Answer: ', answer3);
+question3(answer3);
+
+let answer4 = prompt('Have I ever taken coding classes before? Yes or No or y/n.').toLowerCase();
+answer4 = askAgain(answer4);
+console.log('Q4 Answer: ', answer4);
+question4(answer4);
+
+let answer5 = prompt('Do you think I got married by the water? Yes or No or y/n.').toLowerCase();
+answer5 = askAgain(answer5);
+console.log('Q5 Answer: ', answer5);
+question5(answer5);
+
+let answer6 = 4;
+let guessCount = 0;
+console.log(`Q6 answer ${answer6}`);
+let guess6 = prompt('I know you are getting tired of this, but I need you to guess a number between 1 and 10. You get 4 guesses to get it right.');
+checkGuess(guess6);
+
+bakedGoods();
+
+tally(count);
+
+function question1(answer1){
```

Comment:
Since we stored all the answers as global variables, there's not really much of a reason to make those variables an argument since they're question specific. Your function would work exactly the same without requiring the answer1 argument.